src/ripple/app/paths/impl/BookStep.cpp

```diff
@@ -52,9 +52,12 @@ class BookStep : public StepImp<TIn, TOut, BookStep<TIn, TOut>>
     Book book_;
     AccountID strandSrc_;
     AccountID strandDst_;
-    // Charge transfer fees whan the prev step redeems
+    // Charge transfer fees when the prev step redeems
     Step const* const prevStep_ = nullptr;
     bool const ownerPaysTransferFee_;
+    bool const removeFullyConsumedOffers_;
```

Comment:
If we make sure amendmentRIPD1298 is active before the offer x-ing amendment we don't need `removeFullyConsumedOffers_`.  That switch should turn on in 0.40.0 (though I need to change the date before we release).